akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -50,6 +50,10 @@ private[akka] object Shard {
    */
   @SerialVersionUID(1L) final case class EntityStopped(entityId: EntityId) extends StateChange
 
+  @SerialVersionUID(1L) final case object GetCurrentShardState
```

Comment:
this is in `private[akka] object Shard`, I think they should be in `ShardRegions` companion